packages/services-http/test/cloud.ts

```diff
@@ -31,4 +34,12 @@ export class Test extends BasicTest {
     expect(result.args).to.have.ownProperty('foo1')
     expect(result.args).to.have.ownProperty('foo2')
   }
+
+  @test
+  async 'throws an HttpError'() {
+    const method = 'undefined'
+    const url = `${this.baseUrl}/${method}`
+    const result = this.http.fetch(url)
+    expect(result).to.be.rejectedWith(HttpError)
```

Comment:
await is missing - otherwise a false positive will not cause the test to fail

Reply:
added 'await'